### DISCUSSION TITLE
Uploading and adding into the collection

### DISCUSSION THREAD

**Discussion by Zachkes** [metaplex-foundation/metaplex#1984] [Q&A]
Hi there’s, can someone let me know if I can upload or add into the already uploaded and partly minted collection into candy machine? For example I initially uploaded 1000 NFTs, images file names 0-999. Public already minted around 500 of the collection. Can I upload another 1000 NFTs into the same collection starting from image file names 1000-1999? Or I have to upload into a new candy machine and the image file names have to start from 0-999?

Thank you so much to anyone that can answer this.

**Comment by mojalil** [comment]
@Zachkes did you ever find an answer to this question ?
